core/src/trezor/lvglui/scrs/template.py

```
@@ -364,34 +371,40 @@
 
-        # address
-        self.group_address = ContainerFlexCol(
-            self.content_area, self.derive_btn, pos=(0, 8), padding_row=0
-        )
+            # address
+            self.group_address = ContainerFlexCol(
+                self.content_area, self.derive_btn, pos=(0, 8), padding_row=0
+            )
+        else:
+            self.group_address = ContainerFlexCol(
+                self.content_area, self.title, lv.ALIGN.OUT_BOTTOM_LEFT, pos=(0, 40), padding_row=0
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)